app/scripts/modules/amazon/src/function/configure/FunctionBasicInformation.tsx

```diff
@@ -0,0 +1,271 @@
+            <div className={className}>
+              <strong>Your load balancer will be named: </strong>
+              <HelpField id="aws.function.name" />
+              <span>{this.props.app.name.concat('-').concat(values.functionName)}</span>
+              <Field
+                type="text"
+                style={{ display: 'none' }}
+                className="form-control input-sm no-spel"
+                name="functionName"
+              />
+              {errors.functionName && <ValidationMessage type="error" message={errors.functionName} />}
+            </div>
+          )}
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            fastField={false}
+            name="credentials"
+            label="Account"
+            input={props => (
+              <ReactSelectInput
+                {...props}
+                inputClassName="cloudfoundry-react-select"
+                value={values.credentials}
+                stringOptions={accounts.map((acc: IAccount) => acc.name)}
+                clearable={true}
+              />
+            )}
+          />
+        </div>
+        <div className="sp-margin-m-bottom">
+          <RegionSelectField
+            labelColumns={3}
+            component={values}
+            field="region"
+            account={values.credentials}
+            onChange={this.handleRegionChange}
+            regions={regions}
+          />
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            name="functionName"
+            label="Function Name"
+            help={<HelpField id="aws.function.name" />}
+            input={props => <TextInput {...props} />}
+          />
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            name="runtime"
+            label="Runtime"
+            help={<HelpField id="aws.function.runtime" />}
+            input={props => (
+              <ReactSelectInput
+                {...props}
+                inputClassName="cloudfoundry-react-select"
+                stringOptions={availableRuntimes}
+                clearable={true}
+              />
+            )}
+          />
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            name="s3bucket"
+            label="S3 Bucket"
+            help={<HelpField id="aws.function.s3bucket" />}
+            input={props => <TextInput {...props} placeholder="S3 bucket name" />}
+          />
+        </div>
+        {errors.s3bucket && <ValidationMessage type="error" message={errors.s3bucket} />}
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            name="s3key"
+            label="S3 Key"
+            help={<HelpField id="aws.function.s3key" />}
+            input={props => <TextInput {...props} placeholder="object.zip" />}
+          />
+        </div>
+
+        <div className="sp-margin-m-bottom">
```

Comment:
Can we remove all these wrapper divs with `sp-margin-m-bottom`?  I think the form should look OK without them.

Reply:
you are right.
Removed :) Change will appear in the next push.